ui/app/com/gu/recipeasy/controllers/Application.scala

```diff
@@ -18,6 +18,16 @@ import models.CuratedRecipeForm._
 
 class Application(override val wsClient: WSClient, override val conf: Configuration, db: DB, val messagesApi: MessagesApi) extends Controller with AuthActions with I18nSupport with StrictLogging {
 
+  val AdminWithClearanceUsers: List[String] = List(
+    "alastair.jardine@guardian.co.uk",
+    "nathan.good@guardian.co.uk",
+    "pascal.honore@guardian.co.uk"
```

Comment:
Yep, we are currently setting up a google group to grant the access. Update tomorrow....